stl/inc/mutex

```diff
@@ -32,18 +32,18 @@ _EXPORT_STD class condition_variable_any;
 
 class _Mutex_base { // base class for all mutex types
 public:
-#ifdef _ENABLE_CONSTEXPR_MUTEX_CONSTRUCTOR
+#ifdef _DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR
+    _Mutex_base(int _Flags = 0) noexcept {
+        _Mtx_init_in_situ(_Mymtx(), _Flags | _Mtx_try);
+    }
+#else // ^^^ _DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR / !_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR vvv

```

Comment:
After #4000 merged on 2023-08-30, #3950 merged on 2023-09-21 did more than add arrows - it added discipline regarding the difference between `#if` and `#ifdef`. That is, `!_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR` is not really correct - the region should be properly identified as `!defined(_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR)`.